getting_started/workflow/assets/import_process.rst

```diff
@@ -54,10 +54,10 @@ Additionally, extra assets will be preset in the hidden res://.import folder:
 
 .. image:: img/asset_workflow5.png
 
-If any of the files present in this folder is erased (or the whole folder), the
+If any of the files present in this folder are erased (or the whole folder), the
 asset or assets will be reimported automatically. As such, committing this folder
-to the version control system is optional. It can save time on
-reimporting time when checking out in another computer, but it takes considerably
+to the version control system is optional. It can save time when
+reimporting or checking out on another computer, but it takes considerably
```

Comment:
The previous sentence had wrong grammar but the new one no longer has the correct meaning. What this sentence aims to say is:
- You go to a new computer
- You clone the git repository, which does not have the `.import/` folder
- When you open the project the first time, all assets will have to be reimported, which takes time

Or alternatively, if the `.import/` is committed to Git, you no longer need to reimport as the imported files have been cloned to the new computer. But they will take a lot of space in the Git repository.